src/Infrastructure.EntityFramework/Configurations/TaxRateEntityTypeConfiguration.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Bit.Infrastructure.EntityFramework.Models;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace Bit.Infrastructure.EntityFramework.Configurations;
+
+public class TaxRateEntityTypeConfiguration : IEntityTypeConfiguration<TaxRate>
+{
+    public void Configure(EntityTypeBuilder<TaxRate> builder)
+    {
+        builder
+            .HasIndex(tr => new { tr.Country, tr.PostalCode })
+            .HasFilter($"{nameof(TaxRate.Active)} = 1")
```

Comment:
This is more likely to be fine, but again, I think cloud only so I'd be alright leaving it out instead.

Reply:
Decided to go ahead and remove the indexes that required custom filters.